composer.json

```diff
@@ -96,7 +96,7 @@
         "fabpot/php-cs-fixer": "^1.11",
         "lakion/api-test-case": "^1.0@dev",
         "lakion/mink-debug-extension": "^1.0.1",
-        "malukenho/kawaii-gherkin": "^0.1.2",
+        "malukenho/kawaii-gherkin": "~0.1",
```

Comment:
If `malukenho/kawaii-gherkin` follows semantic versioning I'd rather stay with `^` instead of `~` (`^0.1` is `>=0.1, <0.2`, `~0.1` is `>=0.1, <1.0`).